### DISCUSSION TITLE
[BUG] Add-PnPGroupMember for Everyone except external users

### DISCUSSION THREAD

**Discussion by roberAlb** [pnp/powershell#3817] [General]
### Reporting an Issue or Missing Feature
Add-PnPGroupMember -Identity $Group -LoginName "everyone except external users"

This is working fine for some tenants but not working in some new created tenants. The error I'm getting  **Add-PnPGroupMember : Your organizational policies doesn't allow Everyone except external users to be added to a group site marked as private.**

This is launched into a site collection based in a Private group, but we have other tenants where this is working and we have the same configuration for external sharing.

### Expected behavior 
For us the expected behaviour is that everyone except external users should be added to a site based on a private group as the other tenants.

### Actual behavior 
everyone except external users cannot be added to a site collection based on a private team

### Steps to reproduce behavior
Add-PnPGroupMember -Identity $Group -LoginName "everyone except external users" in  a new tenant

### What is the version of the Cmdlet module you are running?
1.11.0

### Which operating system/environment are you running PnP PowerShell on?
- [X ] Windows

**Comment by Barbarur** [comment]
Is the claim of Everyone Except external users turned on for your affected tenant?

https://pnp.github.io/powershell/cmdlets/Set-PnPTenant.html#example-2

Are you able to add this group manually from browser?
